azure-maven-plugin-lib/src/main/resources/MessageTemplates-aca.yaml

```diff
@@ -0,0 +1,169 @@
+promote:
+  header:       "Available Azure Container App Environments:"
+  many:         "Select Azure Container App Environment for deployment:"
+message:
+  empty_options:  "There are no Azure Container App Environment in current subscription, please create it in Azure Portal."
+  auto_select:    "There is only one Azure Container App Environment (***${ environments[0].name() }***) in current subscription, will use it automatically."
+  select_none:    "You have not selected any Azure Container App Environment, 'config' will terminate."
+required:         true
+auto_select:      true
+default_index:    0
+
+---
+id:               select-app
+promote:
+  header:       "Available Azure Container Apps:"
+  many:         "Select Azure Container App for deployment:"
+message:
+  empty_options:  "There are no Azure Container App in current environment, please create it."
+  auto_select:    "There is only one Azure Container App (***${ apps[0].name() }***) in current environment, will use it automatically."
+  select_none:    "You have not selected any Azure Container App, 'config' will terminate."
+required:         true
+auto_select:      true
+default_index:    0
+
+---
+id:               select-registry
+promote:
+  header:       "Available Azure Container Registries:"
+  many:         "Select Azure Container Registry for deployment:"
+message:
+  empty_options:  "There are no Azure Container Registry in current subscription, please create it."
+  auto_select:    "There is only one Azure Container Registry (***${ registries[0].name() }***) in current subscription, will use it automatically."
+  select_none:    "You have not selected any Azure Container Registry, 'config' will terminate."
+required:         true
+auto_select:      true
+default_index:    0
+
+---
+id:               select-identity
+promote:
+  header:       "Available User Assigned Identities:"
+  many:         "Select User Assigned Identity for deployment:"
+message:
+  empty_options:  "There are no User Assigned Identity  in current subscription, please create it."
+  auto_select:    "There is only one User Assigned Identity (***${ identities[0].name() }***) in current subscription, will use it automatically."
+  select_none:    "You have not selected any User Assigned Identity, 'config' will terminate."
+required:         true
+auto_select:      true
+default_index:    0
+
+---
+id:               configure-app-name
+promote:          "Input the app name (***${default_val}***):"
+resource:         App
+property:         appName
+default_val:          '${project.getArtifactId().replaceAll("[^-a-z0-9]", "").toLowerCase()}'
+expression:       true
+required:         true
+
+---
+id:               configure-deployment-type
+promote:          "Input deployment type for apps (***${schema.default}***):"
+resource:         App
+property:         deploymentType
+
+---
+id:               configure-code-dir
+promote:          "Input directory for source code (***${schema.default}***):"
```

Comment:
It seems the default value is pointing to the Maven plugin source folder instead of the customer's project root. For example, I see `/home/menxiao/Code/azure-maven-plugins/azure-maven-plugin-lib` default value.

I'm wondering if we need to provide this in the config step. I think just take the default source root would be sufficient?

Reply:
The config process use an absolute path as default value, which gives the impression to customer that we need to use absolute path. However, it doesn't make sense to specify an absolute path at all. E.g., I specified `/home/menxiao/tmp/demo`. However, this is likely to change when this is pushed to upstream, and then cloned by other people to other places.

I think we should skip this configuration. The directory should be default to project root. And if the customer needs to update, they can refer to our documentation and update in the pom file.